phpcount.php

```diff
@@ -135,8 +132,8 @@ public static function GetHits($pageID, $unique = false)
         }
         else
         {
-            die("Missing hit count from database!");
-            return false;
+            //die("Missing hit count from database!");
+            return 0;
```

Comment:
There's no reason this should be 0 instead of false, I think it should actually throw an exception, but I won't let that block this PR.